### DISCUSSION TITLE
22.4.49

### DISCUSSION THREAD

**Discussion by immauss** [immauss/openvas#294] [Announcements]
Another minor release from Greenbone, and the addition of a news updates to the login page. Thoughts? Let us know in the comments here. 

-Scott

<hr /><em>This discussion was created from the release <a href='https://github.com/immauss/openvas/releases/tag/22.4.49'>22.4.49</a>.</em>

**Comment by cybermcm** [comment]
@immauss: Thanks for your work (as always). Is there a reason why the latest tag currently isn't the latest image? I use 'latest' for most of my containers (knowing that things can break).

**Comment by immauss** [comment]
22.4.48 wasn't ... because of the db update. But this one should be. I'll double-check . . . . .

:(

The build didn't push out ...

Just re-ran it .. so it should be latest before too long.

Thanks for keeping me honest @cybermcm 

-Scott
